toolchain-linux.cmake

```diff
@@ -5,13 +5,12 @@
 
 if (DEFINED ENV{ROCM_PATH})
   set(rocm_bin "$ENV{ROCM_PATH}/bin")
+  set(llvm_bin "$ENV{ROCM_PATH}/llvm/bin")
 else()
   set(rocm_bin "/opt/rocm/bin")
+  set(llvm_bin "/opt/rocm/llvm/bin")
 endif()
 
 
-# set(CMAKE_CXX_COMPILER "hipcc")
-# set(CMAKE_C_COMPILER "hipcc")
-set(CMAKE_CXX_COMPILER "${rocm_bin}/hipcc")
-set(CMAKE_C_COMPILER "${rocm_bin}/hipcc")
-
+set(CMAKE_CXX_COMPILER "${llvm_bin}/clang++")
```

Comment:
Does env CXX override this?   As discussed at meeting can users easily fallback to hipcc if there are issues in custom build.

Reply:
This is an excellent point, no this is _not_ overridden by CXX. Fixing this momentarily.